tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/featured_case_study.html

```diff
@@ -0,0 +1,53 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% image featured_case_study_image fill-828x546 format-webp as featured_desktop_image %}
+{% image featured_case_study_image fill-1656x1092 format-webp as featured_desktop_image_retina %}
+{% image featured_case_study_image fill-860x646 format-webp as featured_mobile_image %}
+{% image featured_case_study_image fill-430x323 format-webp as featured_mobile_image_retina %}
+
+<div class="featured-case-study">
+    {# Image (with gradient if logo added) #}
+    <div class="featured-case-study__image-wrap{% if logo %} featured-case-study__image-wrap--with-logo{% endif %}">
```

Comment:
I'm not seeing the gradient in the pattern library - probably need a `logo: True` in the yaml

Reply:
Ah thanks. I updated it from `logo` in a previous commit so it wasn't so generic and must have missed this one. Fixed in https://github.com/torchbox/torchbox.com/pull/60/commits/5e45263074e4a3e46860f5d8f80bb6aa4fecea79